setup.py

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env python2.7
 
 import os
-import sys
 import shutil
+
 from setuptools import setup, Extension
 from setuptools.command.install import install
 
 from src.version import softwareVersion
 
+
+EXTRAS_REQUIRE = {
+    'gir': ['pygobject'],
+    'notify2': ['notify2'],
+    'pyopencl': ['pyopencl'],
+    'prctl': ['python_prctl'],  # Named threads
```

Comment:
Where are you going to use this feature?

Reply:
prctl is needed so that the OS can see thread names. If you run `top -H`, it will show thread names assuming prctl is installed, otherwise all will be called `python` or `pybitmessage`.

Reply:
OK, I'm reading up, making sure I understand what your needs are here and considering a solution.

Reply:
OK, I'm reading up, making sure I understand what your needs are here and considering a solution.

Reply:
I have opened issue#1269 to address this.